src/PaymentGateways/PayPalCommerce/PayPalCommerce.php

```diff
@@ -84,19 +83,22 @@ public function getPaymentMethodLabel(): string
      */
     public function createPayment(Donation $donation, $gatewayData): GatewayCommand
     {
-        $command = PaymentComplete::make($gatewayData['paypalOrder']->payment->id);
-        $command->paymentNotes = [
-            sprintf(
-                __('Transaction Successful. PayPal Transaction ID: %1$s    PayPal Order ID: %2$s', 'give'),
-                $gatewayData['paypalOrder']->payment->id,
-                $gatewayData['paypalOrder']->id
-            )
-        ];
+        /** @var PayPalOrder $paypalOrder */
+        $paypalOrder = $gatewayData['paypalOrder'];
```

Comment:
@jonwaldstein What reason to create a variable?

Reply:
Whenever we are referring to an item in an array like this more than once, it starts to become more delicate because we're referencing it's key position as a _string_ every-time.  Storing in a variable ensures we are always referring to the same item in the array.  I like to think if this key changed or the shape of the `gatewayData` changes, we only have to update it in one place.  If there were more items in this array, I would even consider creating a DTO that turns the whole thing into an object so getting the position by string key only happens once for each item.